### PR TITLE
fix jembi integration

### PR DESCRIPTION
**Pull request by miltontony**
- [x] Add better logging of http errors
- [x] stop sending totals and only send per clinic data

### CODE REVIEW

**Comment by miltontony** [comment]
@KaitCrawford review please

File: setup.cfg [hunk 1]
```diff
@@ -2,6 +2,6 @@
 exclude = ve,docs,nurseconnect/migrations/*
 ignore = F403, F405
 
-[pytest]
+[tool:pytest]
```
Comment (KaitCrawford) [inline setup.cfg:5]: What does this do?

**Review by KaitCrawford — APPROVED** [review]
:+1: 
Just a question for my own learning

File: setup.cfg (hunk 1 above)
Comment (miltontony) [reply setup.cfg:5]: new way of specifying parameters for pytest. They've deprecated the just using `pytest`

**Pull request merged**